src/auth/src/credentials/user_account.rs

```diff
@@ -473,6 +492,129 @@ struct Oauth2RefreshResponse {
     refresh_token: Option<String>,
 }
 
+#[allow(dead_code)]
+pub(crate) mod idtoken {
+    /// Credentials for authenticating with [ID tokens] from a [user account].
+    ///
+    /// This module provides a builder for creating [`IDTokenCredentials`] from
```

Comment:
Omit superfluous words:

```suggestion
    /// This module provides a builder for [`IDTokenCredentials`] from
```